pmm-app/src/pmm-qan/panel/components/Overview/components/MetricColumns/MetricColumns.test.tsx

```diff
@@ -18,13 +19,55 @@ jest.mock('shared/components/helpers/getPmmTheme', () => ({
 }));
 
 describe('TimeMetric::', () => {
-  it('should render filters correct', async () => {
-    mount(<TimeMetric value={10} percentage={10} cnt={10} />);
+  it('should render time metric correct with value', async () => {
+    const { container } = render(<TimeMetric value={10} percentage={10} cnt={10} />);
+    const span = container.querySelector('div > span');
+
+    expect(span?.textContent).toBe(humanize.transform(10, 'time'));
```

Comment:
I think I would prefer to use a raw string for the assertion. (whatever is the result of `humanize.transform`)